themes/mlibrary_new/common/footer.php

```diff
@@ -0,0 +1,25 @@
+	</div><!-- end content -->
+</main>
+	<footer class="site-footer">
+			<div class="container container-medium">
+				<div id="footer-content">
+							<p><a href="">Home</a></p>
```

Comment:
Is `href=""` correct here and in the next line?  Seems out of place.

Reply:
One of the anchors (the Help page) doesn't exist yet so I didn't have them linked out yet. I removed both until I can get the proper anchor tag.

Reply:
That makes sense; it's okay to leave those in if you prefer.  I was mostly considering whether it might have been left over from the original theme or such.